### PR TITLE
chore: added new manifests

### PR DESCRIPTION
**Pull request by hatskier**

### CODE REVIEW

File: packages/on-chain-relayer/relayer-manifests/arbitrumPremia.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "chain": {
+    "name": "Arbitrum One",
+    "id": 42161
+  },
+  "updateTriggers": {
+    "cron": ["30 56 7 * * *", "58 7 * * *", "30 59 7 * * *"]
```
Comment (cehali) [inline packages/on-chain-relayer/relayer-manifests/arbitrumPremia.json:7]: Kuba suggested: `"cron": ["55 * * * *", "30 59 * * * *", "40 59 * * * *", "50 59 * * * *"]`

File: packages/on-chain-relayer/relayer-manifests/staderEthx.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "id": 1
+  },
+  "updateTriggers": {
+    "deviationPercentage": 0.5,
+    "timeSinceLastUpdateInMilliseconds": 21600000
+  },
+  "adapterContract": "0xF3eB387Ac1317fBc7E2EFD82214eE1E148f0Fe00",
+  "dataServiceId": "redstone-primary-prod",
+  "priceFeeds": {
+    "ETHx": "0xFaBEb1474C2Ab34838081BFdDcE4132f640E7D2d"
```
Comment (cehali) [inline packages/on-chain-relayer/relayer-manifests/staderEthx.json:13]: I checked this contract and it has round id = 0

**Review by cehali — APPROVED** [review]
LGTM

File: packages/on-chain-relayer/relayer-manifests/staderEthx.json (hunk 2 above)
Comment (koonopek) [reply packages/on-chain-relayer/relayer-manifests/staderEthx.json:13]: What is wrong with round id = 0

Comment (cehali) [reply packages/on-chain-relayer/relayer-manifests/staderEthx.json:13]: It means that we haven't pushed the price manually for the first time which will make it impossible for the relayer to work

**Review by koonopek — APPROVED** [review]
GJ

**Pull request merged**